File: calico-us-berkeley/2025-fall/final/decibel/templates/decibel.java
import java.io.*;

class Solution {
    /**
     * Return the score of S amplified K times
     * 
     * S: string to amplify
     * K: integer for number of times to amplify
     */
    static int solve(String S, int K) {
        // YOUR CODE HERE
        return -1;
    }
    
    static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);

    public static void main(String[] args) throws IOException {
        int T = Integer.parseInt(in.readLine());
        for (int i = 0; i < T; i++) {
            String[] temp = in.readLine().split(" ");
            String S = temp[0]; 
            int K = Integer.parseInt(temp[1]);
            out.println(solve(S, K));
        }
        out.flush();
    }
}

File: calico-us-berkeley/2025-fall/final/decibel/templates/decibel.py
def solve(S: str, K: int) -> int:
    """
    Return the score of S amplified K times
    
    S: string to amplify
    K: integer for number of times to amplify
    """
    # YOUR CODE HERE
    return 0


def main():
    T = int(input())
    for _ in range(T):
        temp = input().split()
        S, K = temp[0], int(temp[1])
        print(solve(S, K))

if __name__ == '__main__':
    main()
